modules/edit-modes/src/lib/draw-polygon-by-dragging-mode.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import type { ClickEvent, ModeProps } from '../types.js';
+import type { Polygon, FeatureCollection } from '../geojson-types.js';
+import { getPickedEditHandle } from '../utils.js';
+import { DrawPolygonMode } from './draw-polygon-mode';
+
+export class DrawPolygonByDraggingMode extends DrawPolygonMode {
```

Comment:
I'm super-thrilled to see the `DrawPolygonMode` flexible enough to be reused here!